dist/doc/html.md

```diff
@@ -176,6 +176,13 @@ included.
 We use `analytics.js` rather than the newer `gtag.js` as
 [it's faster and supports tasks and plugins](https://github.com/philipwalton/analyticsjs-boilerplate/issues/19#issuecomment-333714370)
 
+Starting with version 8 we, by default, <a href="https://support.google.com/analytics/answer/2763052?hl=en">anonymizse IP addresses</a>.
```

Comment:
anonymize typo here 🤓 
Also - should use markdown style links, not HTML ones I think